test/package.json

```
@@ -0,0 +1,27 @@
+{
+  "name": "@bedrock/redis-test",
+  "version": "1.0.0",
+  "description": "Test suites for bedrock-redis",
+  "type": "module",
+  "scripts": {
+    "test": "node --preserve-symlinks test.js test",
+    "coverage": "c8 --reporter=text --reporter=text-summary npm test",
+    "coverage-ci": "c8 --reporter=lcov npm test",
```

Matching pattern used in other modules:
```suggestion
    "coverage": "c8 npm test",
    "coverage-ci": "c8 --reporter=lcovonly --reporter=text --reporter=text-summary npm test",
```